src/commands/timelinecommands.h

```diff
@@ -571,6 +571,28 @@ class ReplaceCommand : public QUndoCommand
     UndoHelper m_undoHelper;
 };
 
+
+class AlignCLipsCommand : public QUndoCommand
```

Comment:
Please change the case of 'L' in Clips to lower case.

Reply:
Done